docs/orchestrating-elastic-stack-applications/kibana.asciidoc

```diff
@@ -224,14 +220,13 @@ spec:
 ----
 
 [id="{p}-kibana-http-configuration"]
-== HTTP Configuration
+== HTTP configuration
 
 [id="{p}-kibana-http-publish"]
 === Load balancer settings and TLS SANs
 
 By default a `ClusterIP` link:https://kubernetes.io/docs/concepts/services-networking/service/[service] is created and associated with the Kibana deployment.
-You may want to expose Kibana externally with a link:https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer[load balancer].
-In which case you may also want to include a custom DNS name or IP in the self-generated certificate.
+If you want to expose Kibana externally with a link:https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer[load balancer], you have to include a custom DNS name or IP in the self-generated certificate.
```

Comment:
> you have to include a custom DNS name or IP in the self-generated certificate.

You don't have to. But if you don't you will see certificate errors when accessing Kibana in a browser. I think that's why it was worded a little vague in the original. I would still weaken the statement a bit: maybe: "it is recommended to set a custom DNS name or IP"?